projects/humans/skin_animated_humans/protos/CharacterSkin.proto

```diff
@@ -0,0 +1,131 @@
+#VRML_SIM R2021c utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# Skin node representing a human with predefined mesh and appearance.
+# Four different characters are available based on the 'model' field:
+# - "Anthony": a boy,
+# - "Robert": a men,
+# - "Sandra": a woman,
+# - "Sophia": a girl.
+# template language: javascript
+
+PROTO CharacterSkin [
+  field SFVec3f                                           translation 0 0 0
+  field SFRotation                                        rotation    0.57735 0.57735 0.57735 2.0944
```

Comment:
Can't we avoid such an odd default rotation? It looks difficult to understand.

Reply:
No.
Human FBX models should be in Left-Up-Front coordinate system otherwise default mocap animation cannot be played out of the box.
So if we want that the default `CharacterSkin` PROTO is oriented correctly using FLU in ENU worlds we need to use this default rotation.

Reply:
It's a pity...

Reply:
And when parsing FBX models/mocap animations, can't we convert them to FLU (from LUF) so that they look good in Webots? Would that be a good idea?

Reply:
We cannot convert FBX models on the Webots side because we don't know if they need to be converted or not.
For example the salamander FBX is already FLU and doesn't need to be converted.
So we should change the orientation of these 4 humans model in the FBX file.

It would be for sure possible to convert the mocap animation on the controller side.
But it is not just a matter of switching the values and we would need to perform additional computations.

Personally I don't think it is a good idea because we will add unneeded complexity and performance loss on the controller side just to remove this rotation and the conversion will only work for this particular example and library
And I think it is more important to keep compatible FBX and mocap files rather than having a simpler default orientation.

Reply:
I see. The problem we have here is that it seems to be the only PROTO with a default rotation that is not user friendly. Protos with a default rotation of 0 0 1 0 can easily be rotated along the vertical axis by simply changing the angle value. In that case, it is not possible...
Maybe we could set the default value to 0 0 1 0 and compute the actual rotation from that in Javascript instead of using a IS keyword?

Reply:
If the rotation is a template regenator field, then the rotation handles cannot be used to rotate the node.
So using `wbrotation.combine()` would make it just worse as the rotation could not be changed using the handles from the 3D scene.
This is an important rule to prevent too many PROTO regeneration while rotating the object with the handles.